src/adapters/utils/networkUtils.js

```diff
@@ -73,15 +74,15 @@ const nodeSysErrorToStatus = code => {
 };
 
 // Returns dynamic Meta based on Status Code as Input
-const getDynamicMeta = statusCode => {
+const getDynamicErrorType = statusCode => {
```

Comment:
sometime we are calling this with empty statusCode, better to handle it explicitly.